js/dynamic.js

```diff
@@ -18,16 +18,52 @@ function displayNav() {
     humburgerNav.style.color = "#6070ff";
     humburgerNav.innerHTML = '<i class="fa fa-bars" aria-hidden="true"></i>';
   } else {
-    menu.style.display = 'block';
+    menu.style.display = "block";
     humburgerNav.style.color = "white";
     humburgerNav.innerHTML = '<i class="fas fa-times"></i>';
   }
 
-  menu.classList.toggle('menu');
+  menu.classList.toggle("menu");
 }
 
 humburgerNav.addEventListener("click", displayNav);
 
+//data storage in the browser
+
+const btn = document.getElementById("submitBtn");
+const body = document.getElementsByTagName('body')[0];
+
+console.log(localStorage.user);
```

Comment:
- [ ] [OPTIONAL] It is not a big issues but I recommend removing the console logs after the development progress. It may confuse the other devs if they come to see your project 👌😉